translations/strings.yaml

```diff
@@ -383,6 +383,14 @@ serverUnavailableNotification:
   preferencesSubLabel: Get notified when connecting to a server location failed
 
 mobileOnboarding:
+  panelOneTitle: Mozilla VPN
+  panelOneSubtitle: A safe, reliable and fast VPN. Built by the makers of Firefox.
+  panelTwoTitle: Encrypt your activity
+  panelTwoSubtitle: Mozilla VPN keeps your location and browsing safe from eavesdroppers.
+  panelThreeTitle: Protect your privacy
+  panelThreeSubtitle: Route your activity and location through servers in more than 30 countries.
+  panelFourTitle: More security with one tap
+  panelFourSubtitle: Stream, download and game without limits, monthly caps or ISP throttling.
```

Comment:
`throttling` sounds a bit too techy for onboarding, but in case that's a copy issue.

My only concern is that translations will likely need a much more verbose form for it (and "caps"), so let's make sure we don't truncate text unnecessarily (from the screenshots, we should have plenty of space to accommodate more text).

Reply:
Thank you for the review! I’ll check back with UX & content to let them that they can elaborate more in case they wish to do so.